Autopilot/PathManager/Inc/pathStateClasses.hpp

```diff
@@ -0,0 +1,132 @@
+#pragma once
```

Comment:
Putting comments at the top (similar to other files in our repo) will help other members get an idea of the purpose of the file (and also give yourself credit for making it!)